core/templates/pages/exploration-editor-page/statistics-tab/services/learner-action-render.service.spec.ts

```diff
@@ -54,29 +54,29 @@ describe('Learner Action Render Service', function() {
         interaction: {
           id: 'Continue',
           customizationArgs: {
-            buttonText: {value: new SubtitledUnicode('', '')}
+            buttonText: { value: new SubtitledUnicode('', '') }
           }
         }
       })
       .withArgs('stateName2').and.returnValue({
         interaction: {
           id: 'TextInput',
           customizationArgs: {
-            placeholder: {value: new SubtitledUnicode('', '')},
-            rows: {value: 1}
+            placeholder: { value: new SubtitledUnicode('', '') },
+            rows: { value: 1 }
           }
         }
       })
       .withArgs('stateName3').and.returnValue({
         interaction: {
           id: 'MultipleChoiceInput',
           customizationArgs: {
-            choices: {value: [
+            choices: { value: [
               new SubtitledHtml('Choice1', ''),
               new SubtitledHtml('Choice2', ''),
               new SubtitledHtml('Choice3', '')
-            ]},
-            showChoicesInShuffledOrder: {value: true}
+            ] },
```

Comment:
@seanlip Can we merge this PR directly as all the changes look mechanical and this PR is prone to conflict (because of changes in 333 files)?

Reply:
Update -- I had a discussion with @DubeySandeep about this and the lint check implemented here might not be fully correct. @DubeySandeep will follow up.

Reply:
Hi @seanlip , @DubeySandeep, Can you please guide me what should I do next? I have updated the lint errors which were showing in the previous review.

Reply:
> @Mayank-gaur We are not expecting to have the following structure in the codebase:
> 
> ```
> let someVaraible = { someKey: [
>   listValue1, listValue1
> ] }
> ```
> 
> The space between `]` & `}` is not expected. I think it's happening because the dict starts as a one-liner-dict but is branched off in multiple lines!
> 
> As per my discussion with core maintainers, we are expecting the following pattern in the codebase:
> 
> ```
> one-liner-dict:
> let someVaraible = { someKey: someValue };
> let someAnotherVaraible = { someKey: [someValues] };
> 
> 
> Multiline dict:
> let someVaraible = { 
>   someKey: someValue 
> };
> ```
> 
> i.e, one-liner-dict should start and end in the same line and should have space before and after key-value.
> A multiline dict should have a newline after `{` and before `}`
> 
> Can you find some lint check which supports this behavior? (Let me know if you have any questions related to the expected behavior?)

Yes @DubeySandeep I have understood the task. I am unable to find any existing lint check achieving this so I think we have to create a custom extension. I can visit every 'ObjectExpression' node and  then I can implement this functionality.  Please guide me wether I should proceed with this approach or not. @DubeySandeep PTAL.